### ISSUE TITLE
MultiDay event Issue

### ISSUE THREAD

**Issue by SachinTanpure** [richardtop/CalendarKit#155]
### New Issue Checklist

- [x] Use the latest version of CalendarKit
- [x] I've read the [Contribution Guidelines](https://github.com/richardtop/CalendarKit/blob/master/CONTRIBUTING.md)
- [x] I've searched for [existing GitHub issues](https://github.com/richardtop/CalendarKit/issues)

### Issue Description
@richardtop First of all thanks for such great library. Currently i am facing one issue regarding multi day event. I have one event which start at 20-May-2018 12:00 AM to  22-May-2018 11:59 PM. So issue is app is showing 3 events on every day(Please check below screen shot). 
![simulator screen shot - iphone 5s - 2018-05-10 at 12 29 09](https://user-images.githubusercontent.com/6615247/39856533-f2c586bc-544d-11e8-961b-5efeca9586fc.png)

Actually it need to show one event per day instead of 3 events per day.

**Comment by richardtop** [comment]
Hi, @SachinTanpure, thank you for your report. It's clearly a bug.

**Comment by richardtop** [comment]
@SachinTanpure thank you!
As far, as I can tell, the delegate method gets called 3 times (as there are 3 pages in the CalendarKit view). All of them are "correct" since they are valid on a given day.

That's definitely a bug in the CalendarKit, as the events should be filtered out correctly.

To temporarily fix it at the app level, try to return multi-day event only once, while there are no fix for this issue in the library.

**Comment by SachinTanpure** [comment]
Okay @richardtop
I hope this issue is going to be resolve soon.
Thank you!

**Comment by tarangpatel** [comment]
@richardtop Does this Calendar support recurrences of event like RRule ?

**Comment by richardtop** [comment]
@tarangpatel No, it is an UI-only component. Any additional logic should be implemented in a custom module providing data to the calendar.

**Comment by richardtop** [comment]
Resolved in the latest version.
